src/app/components/LanguageAvatar.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+  export let name: string;
+  export let selectedLanguage: string = "";
+
+  let selected: boolean = name === selectedLanguage;
+  let className: string = "avatar";
+
+  /**
+   * Returns the first letter / character of the language.
+   * TODO: Replace with a method that handles abstract cases
+   *
+   * @return {String} - the first character of the language name
+   */
+  const getLanguageShorthand = (): String => {
+    return name.substring(0, 1);
+  };
+
+  /**
+   * Handles the click when a language is selected from the sidebar.
+   * Should change the selected language variable and change content in the Languages component.
+   * TODO: Replace stub
+   *
+   * @param {string} languageName - A the name of the language selected.
+   * @return {void}
+   */
+  const handleClick = (languageName: string): void => {
+    console.log(languageName + " button selected");
+  };
+</script>
+
+<style>
+  .avatar {
+    width: 50px;
+    height: 50px;
+    display: flex;
+    background-color: var(--white);
+    align-items: center;
+    justify-content: center;
+    border-radius: 100%;
+    margin-top: 40px;
+    font-family: var(--main-font), sans-serif;
+    font-size: 20px;
+    box-sizing: border-box;
+    transition: border-color 0.2s ease;
+    border: 3px solid var(--white);
+  }
+
+  .avatar:hover {
+    cursor: pointer;
+    box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.1);
+    border: 3px solid var(--primary-blue);
+  }
+
+  .selected {
+    background-color: var(--primary-blue);
+    border: 0px;
+    color: var(--white);
+    box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.1);
+  }
+</style>
+
+<div
+  class={className}
+  class:selected
+  on:click={() => {
+    handleClick(name);
+  }}>
+  {getLanguageShorthand()}
+</div>
```

Comment:
I didn't notice this before, but this _should_ be an ordinary `<a>` link to goes to a new project page. Don't do anything now though, since I think that can be handled in a future PR :)